[W10-3] Equipment Manager

racheltanxueqi opened the pull request:
@Dukesun99
@bendymochi
@LiuXuanIan
@e0191632

kylase-learning commented on docs/DeveloperGuide.adoc:
```diff
@@ -811,41 +811,66 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
+|`* * *` |user | delete worklist | remove worklist which indicates work completed and no longer need it for tracking progress
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user | have a worklist where I can store the equipments I am working on | Keep tack of all the equipments that I'm 
```
From your Travis,

```
...
ERROR:docs/DeveloperGuide.adoc:838: trailing whitespace.
ERROR:docs/UserGuide.adoc:122: trailing whitespace.
ERROR:docs/UserGuide.adoc:259: trailing whitespace.
```

Seems like you have a trailing whitespace.

kylase-learning commented:
> @kylase-learning May i know why this documentation failed on appveyor?

You mean Travis right? I can see the Travis build status here. 

They should be the same problem. You will need to run the log to find out what is happening.

kylase-learning commented:
> @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.

```
/home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
        persons.sortByName();
        ^
  symbol:   variable persons
  location: class EquipmentManager
1 error
```
Your code failed to compile.

According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?

dukesun99 commented:
> > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> 
> ```
> /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
>         persons.sortByName();
>         ^
>   symbol:   variable persons
>   location: class EquipmentManager
> 1 error
> ```
> Your code failed to compile.
> 
> According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?

Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.

kylase-learning commented:
> > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > 
> > 
> > ```
> > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> >         persons.sortByName();
> >         ^
> >   symbol:   variable persons
> >   location: class EquipmentManager
> > 1 error
> > ```
> > Your code failed to compile.
> > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> 
> Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.

This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed? 

Otherwise, provide me the link and let me look at it.

dukesun99 commented:
> > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > 
> > > 
> > > ```
> > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > >         persons.sortByName();
> > >         ^
> > >   symbol:   variable persons
> > >   location: class EquipmentManager
> > > 1 error
> > > ```
> > > Your code failed to compile.
> > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > 
> > 
> > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> 
> This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> 
> Otherwise, provide me the link and let me look at it.

For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.

dukesun99 commented:
@kylase-learning 
We also have another problem. Now we want to use gh-pages to publish a webpage. However if I commit to gh-pages it will be overwritten by Travis bot. Could you help to suggest a way out? Thanks.

kylase-learning commented:
## Feedback (Week 7; v1.1)
 
### Github Project Page
- [ ] Update Project description
- [ ] Update Link to Project Page
 
### README
- [x] UI Mockup updated in project website
- [ ] Make the project website look like a product home page (aka remove the top nav bar)
- [x] AB4 acknowledgment
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's
 
### [Project Page](https://cs2103-ay1819s2-w10-3.github.io/main/)
- [x] Update to remove AB4 reference
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's

### About Us Page
- [x] Updated with photos (_You can remove Prof. from your team_)
- [x] Roles and components assigned

UG/DG feedback will be provided to you in the documents.

kylase-learning commented:
> @kylase-learning
> We also have another problem. Now we want to use gh-pages to publish a webpage. However if I commit to gh-pages it will be overwritten by Travis bot. Could you help to suggest a way out? Thanks.

Don't commit directly to `gh-pages`. If you look at the gradle script, it basically builds the documentation from the `/docs` folder and push it to your `gh-pages` branch.

Therefore, you make all the documentation changes in your branches that are derived from `master`.

Hope this clarifies. Let me know if it doesn't.

kylase-learning commented:
> > > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > > 
> > > > 
> > > > ```
> > > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > > >         persons.sortByName();
> > > >         ^
> > > >   symbol:   variable persons
> > > >   location: class EquipmentManager
> > > > 1 error
> > > > ```
> > > > Your code failed to compile.
> > > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > > 
> > > 
> > > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> > 
> > 
> > This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> > Otherwise, provide me the link and let me look at it.
> 
> For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
> For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.

The Travis link you provided shows me the build passed. Maybe if you want to come to my office and resolve this if you can't reconcile the issue.

dukesun99 commented:
> > > > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > > > 
> > > > > 
> > > > > ```
> > > > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > > > >         persons.sortByName();
> > > > >         ^
> > > > >   symbol:   variable persons
> > > > >   location: class EquipmentManager
> > > > > 1 error
> > > > > ```
> > > > > Your code failed to compile.
> > > > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > > > 
> > > > 
> > > > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> > > 
> > > 
> > > This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> > > Otherwise, provide me the link and let me look at it.
> > 
> > 
> > For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
> > For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.
> 
> The Travis link you provided shows me the build passed. Maybe if you want to come to my office and resolve this if you can't reconcile the issue.

Just now I reran the Travis test and then it gave me pass. So it is quite random. If I run Travis on the same build multiple time, it will give different results.
This one is failing as well: https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504387259?utm_source=github_status&utm_medium=notification

kylase-learning commented:
> > > > > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > > > > 
> > > > > > 
> > > > > > ```
> > > > > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > > > > >         persons.sortByName();
> > > > > >         ^
> > > > > >   symbol:   variable persons
> > > > > >   location: class EquipmentManager
> > > > > > 1 error
> > > > > > ```
> > > > > > Your code failed to compile.
> > > > > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > > > > 
> > > > > 
> > > > > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> > > > 
> > > > 
> > > > This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> > > > Otherwise, provide me the link and let me look at it.
> > > 
> > > 
> > > For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
> > > For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.
> > 
> > 
> > The Travis link you provided shows me the build passed. Maybe if you want to come to my office and resolve this if you can't reconcile the issue.
> 
> Just now I reran the Travis test and then it gave me pass. So it is quite random. If I run Travis on the same build multiple time, it will give different results.
> This one is failing as well: https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504387259?utm_source=github_status&utm_medium=notification

For this build, your test failed.

```
systemtests.DeleteCommandSystemTest > delete FAILED
    java.lang.AssertionError at DeleteCommandSystemTest.java:167
```

You will need to read the log of the Travis to find out what is wrong. The builds sometime can fail due to resource constraints, therefore you will need to understand what fails.

dukesun99 commented:
> > > > > > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > > > > > 
> > > > > > > 
> > > > > > > ```
> > > > > > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > > > > > >         persons.sortByName();
> > > > > > >         ^
> > > > > > >   symbol:   variable persons
> > > > > > >   location: class EquipmentManager
> > > > > > > 1 error
> > > > > > > ```
> > > > > > > Your code failed to compile.
> > > > > > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > > > > > 
> > > > > > 
> > > > > > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> > > > > 
> > > > > 
> > > > > This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> > > > > Otherwise, provide me the link and let me look at it.
> > > > 
> > > > 
> > > > For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
> > > > For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.
> > > 
> > > 
> > > The Travis link you provided shows me the build passed. Maybe if you want to come to my office and resolve this if you can't reconcile the issue.
> > 
> > 
> > Just now I reran the Travis test and then it gave me pass. So it is quite random. If I run Travis on the same build multiple time, it will give different results.
> > This one is failing as well: https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504387259?utm_source=github_status&utm_medium=notification
> 
> For this build, your test failed.
> 
> ```
> systemtests.DeleteCommandSystemTest > delete FAILED
>     java.lang.AssertionError at DeleteCommandSystemTest.java:167
> ```
> You will need to read the log of the Travis to find out what is wrong. The builds sometime can fail due to resource constraints, therefore you will need to understand what fails.

Yes I know delete is failing. But I really do not have ideas about why it is failing randomly. What should we do?

kylase-learning commented:
> > > > > > > > > @kylase-learning Could you please give any advice on why only Travis is failing? Thank you very much.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > ```
> > > > > > > > /home/travis/build/nus-cs2103-AY1819S2/addressbook-level4/src/main/java/seedu/address/model/EquipmentManager.java:102: error: cannot find symbol
> > > > > > > >         persons.sortByName();
> > > > > > > >         ^
> > > > > > > >   symbol:   variable persons
> > > > > > > >   location: class EquipmentManager
> > > > > > > > 1 error
> > > > > > > > ```
> > > > > > > > Your code failed to compile.
> > > > > > > > According to this [line](https://github.com/nus-cs2103-AY1819S2/addressbook-level4/blob/8ea401ab0d54b5e743e825432d677d3c8acbd4a4/src/main/java/seedu/address/model/EquipmentManager.java#L102), I believe that you don't have `persons` anymore. Shouldn't it be `equipment`?
> > > > > > > 
> > > > > > > 
> > > > > > > Yes that one we are solving. Could you please also look at our second last commit (mine one)? That one only Travis is failing. And pass all tests on my computer.
> > > > > > 
> > > > > > 
> > > > > > This build (https://travis-ci.org/nus-cs2103-AY1819S2/addressbook-level4/builds/504387269)? Looks fine to me and all passed?
> > > > > > Otherwise, provide me the link and let me look at it.
> > > > > 
> > > > > 
> > > > > For https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504512556?utm_source=github_status&utm_medium=notification, Travis is failing, AppVeyor is passing.
> > > > > For https://ci.appveyor.com/project/e0191632/main/builds/22970378, AppVeyor is failing, Travis is passing.
> > > > 
> > > > 
> > > > The Travis link you provided shows me the build passed. Maybe if you want to come to my office and resolve this if you can't reconcile the issue.
> > > 
> > > 
> > > Just now I reran the Travis test and then it gave me pass. So it is quite random. If I run Travis on the same build multiple time, it will give different results.
> > > This one is failing as well: https://travis-ci.org/CS2103-AY1819S2-W10-3/main/builds/504387259?utm_source=github_status&utm_medium=notification
> > 
> > 
> > For this build, your test failed.
> > ```
> > systemtests.DeleteCommandSystemTest > delete FAILED
> >     java.lang.AssertionError at DeleteCommandSystemTest.java:167
> > ```
> > You will need to read the log of the Travis to find out what is wrong. The builds sometime can fail due to resource constraints, therefore you will need to understand what fails.
> 
> Yes I know delete is failing. But I really do not have ideas about why it is failing randomly. What should we do?

Most of the time, it should not fail randomly. 

Like I mentioned in the tutorial, you should not merge to your `master` until you are sure there are no issues and bugs. Therefore, you need to pass all the checks in the CI at your own fork first. 

If you are not clear on the workflow, please come and meet me to discuss.

kylase-learning commented on LICENSE:
```diff
@@ -2,11 +2,11 @@ MIT License
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any equipment obtaining a copy
```
I can see that you have used **Find and Replace All** to do refactoring of `person ` to `equipment `😅.

racheltanxueqi replied:
Hahaha opps. ok we will this file equipment back to person

kylase-learning commented on LICENSE:
```diff
@@ -6,7 +6,7 @@ Permission is hereby granted, free of charge, to any person obtaining a copy
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit person to whom the Software is
```
This as well...

You should use the refactoring in IntelliJ to help you instead.

kylase-learning commented on docs/templates/LICENSE:
```diff
@@ -9,7 +9,7 @@ Permission is hereby granted, free of charge, to any person obtaining a copy
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit person to whom the Software is
```
Same issue.

kylase-learning commented on src/main/java/seedu/equipment/model/equipment/SerialNumber.java:
```diff
@@ -0,0 +1,58 @@
+     * The first character of the serial number must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```
You might want to test this properly.

dukesun99 committed:
Fix style error.

LiuXuanIan committed:
Update on UserGuide and PPP.

dukesun99 committed:
Remove trailing spaces in DisplayEquipmentDetail

LiuXuanIan committed:
Merge pull request #99 from CS2103-AY1819S2-W10-3/edit_docs

Update on UserGuide and PPP.

LiuXuanIan committed:
Merge pull request #98 from CS2103-AY1819S2-W10-3/add-equipment-detail-page

Add equipment detail page

dukesun99 committed:
Update Yiqun's PPP and About US page

dukesun99 committed:
Fix hyperlinks in Yiqun's PPP

racheltanxueqi committed:
Update userguide for list-c

dukesun99 committed:
Merge pull request #100 from CS2103-AY1819S2-W10-3/update-PPP

Update ppp

bendymochi committed:
Update PPP

bendymochi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into docs

bendymochi committed:
Added more details and fix checkstyle

racheltanxueqi committed:
add client panel

bendymochi committed:
Fix checkstyle

bendymochi committed:
Fix checkstyle

dukesun99 committed:
Fix reference issues in PPP

racheltanxueqi committed:
remove ClientListPanel execution in MainWindow

racheltanxueqi committed:
fix checkstyle

racheltanxueqi committed:
Merge branch 'master' into client-details

dukesun99 committed:
Merge pull request #102 from CS2103-AY1819S2-W10-3/update-PPP-Yiqun

Fix reference issues in PPP

racheltanxueqi committed:
Merge branch 'master' into client-details

bendymochi committed:
Merge pull request #101 from CS2103-AY1819S2-W10-3/docs

Update PPP

bendymochi committed:
Add portfolio in AboutUs.adoc

bendymochi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into docs

racheltanxueqi committed:
fix ppp link to .html

racheltanxueqi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S2-W10-3/main:
  fix ppp link to .html

bendymochi committed:
Fix PPP

dukesun99 committed:
Fix tags in UG

bendymochi committed:
Fix PPP

dukesun99 committed:
Merge branch 'master' into update-PPP-Yiqun

bendymochi committed:
Fix PPP

dukesun99 committed:
Merge pull request #105 from CS2103-AY1819S2-W10-3/update-PPP-Yiqun

Fix tags in UG

dukesun99 committed:
Revert "Fix tags in UG"

dukesun99 committed:
Merge pull request #106 from CS2103-AY1819S2-W10-3/revert-105-update-PPP-Yiqun

Revert "Fix tags in UG"

dukesun99 committed:
Fix tags in UG

dukesun99 committed:
Merge branch 'master' into update-PPP-Yiqun-New

dukesun99 committed:
Merge pull request #107 from CS2103-AY1819S2-W10-3/update-PPP-Yiqun-New

Fix tags in UG

racheltanxueqi committed:
Update UserGuide.adoc

dukesun99 committed:
Add developer guide back

dukesun99 committed:
Merge pull request #108 from CS2103-AY1819S2-W10-3/add-developer-guide-back

Add developer guide back

racheltanxueqi committed:
Update UserGuide.adoc

racheltanxueqi committed:
Update UserGuide.adoc

racheltanxueqi committed:
Update rachel.adoc

racheltanxueqi committed:
Update UserGuide.adoc

racheltanxueqi committed:
Update rachel.adoc

racheltanxueqi committed:
Update rachel.adoc

racheltanxueqi committed:
Update UserGuide.adoc

racheltanxueqi committed:
Update UserGuide.adoc

racheltanxueqi committed:
Merge branch 'client-details' of https://github.com/CS2103-AY1819S2-W10-3/main into client-details

* 'client-details' of https://github.com/CS2103-AY1819S2-W10-3/main:
  Fix reference issues in PPP
  Fix hyperlinks in Yiqun's PPP
  Update Yiqun's PPP and About US page
  Remove trailing spaces in DisplayEquipmentDetail
  Update on UserGuide and PPP.
  Fix style error.
  Update UserGuide.adoc
  Fix line length issue.
  Change the base URL to actual url.
  Now, add-w need one more attribute, id.
  Update equipemnt detail page to support split window.
  Update DeveloperGuide.adoc
  Now, add-w need one more attribute, id.
  Now, add-w need one more attribute, id.
  Now, add-w need one more attribute, id.
  Now, add-w need one more attribute, id.

racheltanxueqi committed:
Merge branch 'master' into client-details

racheltanxueqi committed:
update adoc

racheltanxueqi committed:
Merge branch 'client-details' of https://github.com/CS2103-AY1819S2-W10-3/main into client-details

* 'client-details' of https://github.com/CS2103-AY1819S2-W10-3/main: (22 commits)
  Update UserGuide.adoc
  Update UserGuide.adoc
  Update rachel.adoc
  Update rachel.adoc
  Update UserGuide.adoc
  Update rachel.adoc
  Update UserGuide.adoc
  Update UserGuide.adoc
  Add developer guide back
  Update UserGuide.adoc
  Fix tags in UG
  Revert "Fix tags in UG"
  Fix PPP
  Fix PPP
  Fix tags in UG
  Fix PPP
  fix ppp link to .html
  Add portfolio in AboutUs.adoc
  Fix checkstyle
  Fix checkstyle
  ...

LiuXuanIan committed:
Put Command.

LiuXuanIan committed:
Put Command.

racheltanxueqi committed:
Add system tests

LiuXuanIan committed:
Put Command.

racheltanxueqi committed:
Add test in logicmanagertest

racheltanxueqi committed:
fixing tags

racheltanxueqi committed:
fix tags

racheltanxueqi committed:
check style

racheltanxueqi committed:
Add test to Logic Manager Test

racheltanxueqi committed:
Temporary comment out panel for list-c

racheltanxueqi committed:
remove client panel codes from equipmentlistpanel

LiuXuanIan committed:
Put Command.

LiuXuanIan committed:
Put Command.

LiuXuanIan committed:
Put Command.

racheltanxueqi committed:
Merge pull request #103 from CS2103-AY1819S2-W10-3/client-details

List Client details (list-c command)

LiuXuanIan committed:
Put Command.

LiuXuanIan committed:
Merge pull request #109 from CS2103-AY1819S2-W10-3/put_Command

Put command

racheltanxueqi committed:
remove data encryption in implementation section

racheltanxueqi committed:
Add UI section in DG
Edit Quick start in UG, need to update GUI png with correct labelling

racheltanxueqi committed:
Add outline for  equipment and client feature in DG

bendymochi committed:
Merge pull request #111 from CS2103-AY1819S2-W10-3/update-dg-ug

Add UI, Equipment, Client feature in DG

racheltanxueqi committed:
add client panel

racheltanxueqi committed:
change dimension of client panel and worklist panel

racheltanxueqi committed:
Selection of client won't select the equipment together

racheltanxueqi committed:
fix list-c, will only update client panel when list-c

racheltanxueqi committed:
adding and equipment will reflect distinct client's name on client panel

racheltanxueqi committed:
fix checkstyle
fix error in client panel

bendymochi committed:
Refactor Email to Date

bendymochi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into filter-feature

racheltanxueqi committed:
add add-e, delete-e in equipment panel sync with client panel

LiuXuanIan committed:
Merge pull request #112 from CS2103-AY1819S2-W10-3/filter-feature

Refactor Email to Date

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

racheltanxueqi committed:
Add Client Details Panel
Add labels to the 3 different panels
Add add-c command
Add testing for add-c command
fix checkstyle

LiuXuanIan committed:
Merge pull request #115 from CS2103-AY1819S2-W10-3/client-panel

Add Client Panel, Add add-c commands and testing

LiuXuanIan committed:
Modify WorkList ui.

LiuXuanIan committed:
Modify WorkList ui.

racheltanxueqi committed:
Merge pull request #114 from CS2103-AY1819S2-W10-3/worklist_ui_modification

Modify some UI part regarding the WorkList

racheltanxueqi committed:
Update Rachel PPP
Update userguide

racheltanxueqi committed:
Update PPP and userguide

racheltanxueqi committed:
Merge pull request #175 from CS2103-AY1819S2-W10-3/client-panel

Update UG and PPP for client panel feature

bendymochi committed:
Update DG and UG

bendymochi committed:
Update DG

bendymochi committed:
Update UG

bendymochi committed:
Update PPP

dukesun99 committed:
Add DisplayRoutes.html to static pages to publish

dukesun99 committed:
Beautify format

bendymochi committed:
Merge pull request #176 from CS2103-AY1819S2-W10-3/filter-feature

Update DG and UG

dukesun99 committed:
Merge pull request #177 from CS2103-AY1819S2-W10-3/create-static-pages-for-routing

Create static pages for routing

racheltanxueqi committed:
update ug and dg
update ui png
update ui with labels
update and add more ss, uml diagrams

racheltanxueqi committed:
add ui png

dukesun99 committed:
Implement route function

dukesun99 committed:
Add tests for route feature and fix bug

bendymochi committed:
edit SortCommand to sort by date and change date format

bendymochi committed:
Merge pull request #178 from CS2103-AY1819S2-W10-3/filter-feature

edit SortCommand to sort by date and change date format

dukesun99 committed:
Add unit test for route command class.

dukesun99 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into implement-route-feature

dukesun99 committed:
Fix date format in tests

dukesun99 committed:
Add more tests on RouteCommand

racheltanxueqi committed:
Update on DG

racheltanxueqi committed:
update dg ug

racheltanxueqi committed:
Merge branch 'master' into client-panel

racheltanxueqi committed:
Merge pull request #180 from CS2103-AY1819S2-W10-3/client-panel

Update dg and ug

dukesun99 committed:
Fix checkstyle

dukesun99 committed:
Fix tests.

dukesun99 committed:
Fix style.

dukesun99 committed:
Merge pull request #179 from CS2103-AY1819S2-W10-3/implement-route-feature

Implement route feature

LiuXuanIan committed:
remove command added.

LiuXuanIan committed:
PutCommand system test and RemoveCommand system test added.

LiuXuanIan committed:
PutCommand system test and RemoveCommand system test added.

LiuXuanIan committed:
PutCommand system test and RemoveCommand system test added.

bendymochi committed:
Merge pull request #181 from CS2103-AY1819S2-W10-3/removeCommand

remove command added.

LiuXuanIan committed:
deleteCommand added.

LiuXuanIan committed:
deleteCommand added.

LiuXuanIan committed:
deleteCommand added.

racheltanxueqi committed:
Merge pull request #182 from CS2103-AY1819S2-W10-3/deleteCommand

Delete command

racheltanxueqi committed:
Add select-c to select client

racheltanxueqi committed:
fix select command, find selected client by filter function

racheltanxueqi committed:
remove system print codes

racheltanxueqi committed:
Added selectClientCommand test

racheltanxueqi committed:
Add testing for select client command

racheltanxueqi committed:
space

racheltanxueqi committed:
Fix client cards

racheltanxueqi committed:
remove add client command and testing

racheltanxueqi committed:
Enhance UI Labels

racheltanxueqi committed:
fix checkstyle

racheltanxueqi committed:
update dg, all to update the command summary accordingly

racheltanxueqi committed:
update dg

racheltanxueqi committed:
update ug, ppp

dukesun99 committed:
Fix bug related to #169 select-display-select bug

The bug is when select 1, display, and select 1, the browser panel will not change. I fix this in this commit. I add more tests to deal with it aslo.

dukesun99 committed:
Add more test to route related to the #169 issue

dukesun99 committed:
Merge pull request #186 from CS2103-AY1819S2-W10-3/fix-select-display-select-bug

Fix select display select bug

racheltanxueqi committed:
Update ppp

dukesun99 committed:
Merge branch 'master' into client-equipments

 Conflicts:
	src/main/java/seedu/equipment/model/Model.java
	src/test/java/seedu/equipment/logic/commands/AddClientCommandTest.java

 Changes to be committed:
	modified:   src/main/java/seedu/equipment/logic/commands/DisplayCommand.java
	modified:   src/main/java/seedu/equipment/logic/commands/RouteCommand.java
	modified:   src/main/java/seedu/equipment/model/Model.java
	modified:   src/main/java/seedu/equipment/model/ModelManager.java
	modified:   src/main/java/seedu/equipment/ui/BrowserPanel.java
	modified:   src/test/java/seedu/equipment/logic/commands/AddCommandTest.java
	modified:   src/test/java/seedu/equipment/logic/commands/AddWorkListCommandTest.java
	modified:   src/test/java/systemtests/DisplayCommandSystemTest.java
	modified:   src/test/java/systemtests/RouteCommandSystemTest.java

dukesun99 committed:
Add config file for repoSense

dukesun99 committed:
Add newline

dukesun99 committed:
Merge pull request #187 from CS2103-AY1819S2-W10-3/fix-reposense

Fix reposense

racheltanxueqi committed:
update ug, make it more consistent

racheltanxueqi committed:
update dg

bendymochi committed:
Update UG and DG diagrams

racheltanxueqi committed:
comment out a test

racheltanxueqi committed:
fix checkstyle

racheltanxueqi committed:
fix check style

dukesun99 committed:
Merge remote-tracking branch 'origin/client-equipments'

dukesun99 committed:
Adding extra timeout time as needed

racheltanxueqi committed:
Merge pull request #185 from CS2103-AY1819S2-W10-3/client-equipments

Fix Client Panel and Enhance Equipment Details

racheltanxueqi committed:
Merge branch 'master' into filter-feature

racheltanxueqi committed:
Merge pull request #188 from CS2103-AY1819S2-W10-3/filter-feature

Update UG and DG diagrams

racheltanxueqi committed:
update ug

racheltanxueqi committed:
Update About equipment parameters format

racheltanxueqi committed:
update ug

racheltanxueqi committed:
update ug

racheltanxueqi committed:
added images and diagrams to add-e

racheltanxueqi committed:
update ppp and tags

racheltanxueqi committed:
update ui and command prompt

racheltanxueqi committed:
Merge pull request #189 from CS2103-AY1819S2-W10-3/update-dg-ug

Update DG, UG, PPP

dukesun99 committed:
Update UG, DG for display and route

dukesun99 committed:
Update user stories

dukesun99 committed:
Update PPP

dukesun99 committed:
Update PPP

dukesun99 committed:
Update PPP

bendymochi committed:
Fix typos in UG and DG and update PPP

bendymochi committed:
Change size of image

bendymochi committed:
Merge pull request #190 from CS2103-AY1819S2-W10-3/filter-feature

Fix typos in UG and DG and PPP

bendymochi committed:
Change title from Address App to Equipment Manager

bendymochi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into filter-feature

racheltanxueqi committed:
change application icon

bendymochi committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main into filter-feature

bendymochi committed:
Refactor all persons to equipment and addressBook to equipmentManager

bendymochi committed:
Fix checkstyles

bendymochi committed:
update PPP and aboutus

racheltanxueqi committed:
fix client panel

racheltanxueqi committed:
Fix duplicated client names in client panel

racheltanxueqi committed:
Fix checkstyle

racheltanxueqi committed:
fix check style

LiuXuanIan committed:
Modify ui.

LiuXuanIan committed:
Modify ui.

LiuXuanIan committed:
Modify ui.

LiuXuanIan committed:
Modify ui.

racheltanxueqi committed:
Merge pull request #192 from CS2103-AY1819S2-W10-3/equipment-details

Fix Client panels

racheltanxueqi committed:
fix clear command to work on client panel

racheltanxueqi committed:
fix checkstyle

racheltanxueqi committed:
Merge pull request #194 from CS2103-AY1819S2-W10-3/equipment-details

Fix clear command to work on client panel

LiuXuanIan committed:
Merge pull request #193 from CS2103-AY1819S2-W10-3/deleteCommand

auto-order by worklist id

racheltanxueqi committed:
Merge branch 'master' into filter-feature

bendymochi committed:
Solve conflict merge and rename back to persons in MIT

bendymochi committed:
Merge branch 'filter-feature' of https://github.com/CS2103-AY1819S2-W10-3/main into filter-feature

LiuXuanIan committed:
Merge pull request #191 from CS2103-AY1819S2-W10-3/filter-feature

Refactor persons to equipment and addressBook to equipmentManager

racheltanxueqi committed:
Update PPP, DG, UG

racheltanxueqi committed:
fix dg sequence diagram

dukesun99 committed:
Update use cases

racheltanxueqi committed:
add a selectclient use case

racheltanxueqi committed:
update PPP

dukesun99 committed:
Modify display sequence digram

racheltanxueqi committed:
update PPP, reposense link

dukesun99 committed:
Update diagrams for route, display

racheltanxueqi committed:
Update Appendix F manual testing

racheltanxueqi committed:
Merge pull request #195 from CS2103-AY1819S2-W10-3/final-documentation-update

Final Documentation update - Rachel

dukesun99 committed:
Add screenshots for UG

dukesun99 committed:
Update PPP and ug tag

dukesun99 committed:
Update PPP

bendymochi committed:
Update manual testing for sort and filter command

racheltanxueqi committed:
Update DeveloperGuide.adoc

dukesun99 committed:
Merge branch 'master' into update-docs-Yiqun

racheltanxueqi committed:
Update rachel.adoc

bendymochi committed:
Merge pull request #197 from CS2103-AY1819S2-W10-3/filter-feature

Update manual testing for sort and filter command in DG

dukesun99 committed:
Merge pull request #196 from CS2103-AY1819S2-W10-3/update-docs-Yiqun

Update docs yiqun

bendymochi committed:
Resize image in UG

bendymochi committed:
Resize image in UG

bendymochi committed:
Change image size in UG

dukesun99 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W10-3/main

dukesun99 committed:
Update UG images

bendymochi committed:
Change image sizes in UG

dukesun99 committed:
Merge pull request #198 from CS2103-AY1819S2-W10-3/Yiqun-update-images

Yiqun update images.

dukesun99 committed:
Update PPP

LiuXuanIan committed:
Solve bugs and update docs.

bendymochi committed:
Update PPP

bendymochi committed:
Merge pull request #199 from CS2103-AY1819S2-W10-3/filter-feature

update PPP

LiuXuanIan committed:
Solve bugs and update docs.

racheltanxueqi committed:
Rename rachel.adoc to e0191632.adoc

racheltanxueqi committed:
Update AboutUs.adoc

bendymochi committed:
Update AboutUs.adoc

racheltanxueqi committed:
Update DeveloperGuide.adoc

racheltanxueqi committed:
Update e0191632.adoc

LiuXuanIan committed:
Solve bugs and update docs.

LiuXuanIan committed:
Solve bugs and update docs.

LiuXuanIan committed:
Solve bugs and update docs.

LiuXuanIan committed:
Solve bugs and update docs.

racheltanxueqi committed:
Update UserGuide.adoc

dukesun99 committed:
Merge branch 'master' into solve_bugs

LiuXuanIan committed:
Merge pull request #200 from CS2103-AY1819S2-W10-3/solve_bugs

Solve bugs

LiuXuanIan committed:
Rename LiuXuan.adoc to LiuXuanIan.adoc

LiuXuanIan committed:
Update AboutUs.adoc

LiuXuanIan committed:
Update AboutUs.adoc
